### DISCUSSION TITLE
DNS SETUP PROBLM

### DISCUSSION THREAD

**Discussion by pxplilox** [postalserver/postal#2196] [Help with using Postal]
![image](https://user-images.githubusercontent.com/112106679/197237632-181cc797-1280-4c7f-b752-25cbf361d9a6.png)

hello dear igot this issues how can i update it via putty i already replace all example to domain.com via winSCP but still have the problem

**Comment by willpower232** [comment]
if you have updated the config file to remove all example.com's then you need to restart postal to apply the updated config file
